treedoc/tests/test_CLI.py

```diff
@@ -31,6 +31,21 @@ def test_cli_smoketests(obj_string):
     assert exit_code == 0
 
 
+packages = ["numpy", "scipy", "pandas", "sklearn"]
+
+
+@pytest.mark.parametrize("library", packages)
+def test_cli_smoketests_famous_packages(library):
+    """The smoketests assure that the commands run without errors. No output testing
+    is performed apart from the non-existence of an error."""
+
+    args = " ".join(["treedoc", obj_string, "--subpackages"])
```

Comment:
Should this be ` args = " ".join(["treedoc", library, "--subpackages"])`?

Reply:
Very good catch. Did change it :)